server/models/video/video-abuse.ts

```diff
@@ -7,14 +9,164 @@ import {
+    search?: string
+    searchReporter?: string
+    searchVideo?: string
+    searchVideoChannel?: string
+    serverAccountId: number
+    userAccountId: any
+  }) => {
+    let where = {
+      reporterAccountId: {
+        [Op.notIn]: literal('(' + buildBlockedAccountSQL(options.serverAccountId, options.userAccountId) + ')')
+      }
+    }
+
+    if (options.search) {
+      where = Object.assign(where, {
+        [Op.or]: [
+          {
+            [Op.and]: [
+              { videoId: { [Op.not]: null } },
+              { '$Video.name$': { [Op.iLike]: `%${options.search}%` } }
+            ]
+          },
+          {
+            [Op.and]: [
+              { videoId: { [Op.not]: null } },
+              { '$Video.VideoChannel.name$': { [Op.iLike]: `%${options.search}%` } }
+            ]
+          },
+          {
+            [Op.and]: [
+              { deletedVideo: { [Op.not]: null } },
+              { deletedVideo: { name: { [Op.iLike]: `%${options.search}%` } } }
+            ]
+          },
+          {
+            [Op.and]: [
+              { deletedVideo: { [Op.not]: null } },
+              { deletedVideo: { channel: { displayName: { [Op.iLike]: `%${options.search}%` } } } }
+            ]
+          },
+          { '$Account.name$': { [Op.iLike]: `%${options.search}%` } }
+        ]
+      })
+    }
 
+    return {
+      attributes: {
+        include: [
+          [
+            literal(
+              '(' +
+                'SELECT t.count ' +
+                'FROM ( ' +
+                  'SELECT id, ' +
+                         'count(id) OVER (PARTITION BY "videoId") ' +
+                  'FROM "videoAbuse" ' +
+                ') t ' +
+                'WHERE t.id = "VideoAbuseModel".id ' +
+              ')'
+            ),
+            'countReportsForVideo'
+          ],
+          [
+            literal(
+              '(' +
+                'SELECT t.nth ' +
+                'FROM ( ' +
+                  'SELECT id, ' +
+                         'row_number() OVER (PARTITION BY "videoId" ORDER BY "createdAt") AS nth ' +
+                  'FROM "videoAbuse" ' +
+                ') t ' +
+                'WHERE t.id = "VideoAbuseModel".id ' +
+              ')'
+            ),
+            'nthReportForVideo'
+          ],
+          [
+            literal(
+              '(' +
+                'SELECT count("videoAbuse"."id") ' +
+                'FROM "videoAbuse" ' +
+                'INNER JOIN "video" ON "video"."id" = "videoAbuse"."videoId" ' +
+                'INNER JOIN "videoChannel" ON "videoChannel"."id" = "video"."channelId" ' +
+                'INNER JOIN "account" ON "videoChannel"."accountId" = "account"."id" ' +
+                'WHERE "account"."id" = "VideoAbuseModel"."reporterAccountId" ' +
+              ')'
+            ),
+            'countReportsForReporter'
+          ],
+          [
+            literal(
+              '(' +
+                'WITH ' +
```

Comment:
```
SELECT COUNT(*) FROM "videoAbuse" 
INNER JOIN "video" ON "video"."id" = "videoAbuse"."videoId"
INNER JOIN "videoChannel" ON "videoChannel"."id" = "video"."channelId"
INNER JOIN "account" ON "videoChannel"."accountId" = "VideoAbuseModel.Video.VideoChannel.accountId"
```

You'll have to find the real column name generated by sequelize of `"VideoAbuseModel.Video.VideoChannel.accountId"`

Reply:
Actually sequelize has a syntax *for associations*: "Video->VideoChannel"."accountId".